cmd/kube-controller-manager/app/core.go

```diff
@@ -41,6 +45,7 @@ import (
 	resourcequotacontroller "k8s.io/kubernetes/pkg/controller/resourcequota"
 	serviceaccountcontroller "k8s.io/kubernetes/pkg/controller/serviceaccount"
 	ttlcontroller "k8s.io/kubernetes/pkg/controller/ttl"
+	kubectlutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
```

Comment:
I didn't even remember the rest cached rest mapper is in pkg/kubectl. This dependency looks really wrong. If we really want this, we should move the cached rest mapper somewhere else.